usdevs/uscwebsite-hackathon-backend
# Implement User Login logic

Closes #26. This PR adds the logic to update the db when users login. Currently, only users whose TelegramId/TelegramUsername are allowed to login to the app.

To test:
1. Use frontend https://usdevs.github.io/uscwebsite-hackathon-backend/
2. Start the DB with `docker-compose up -d` and do the necessary migrations/seeding
3. Add BOT_TOKEN to `.env` file
4. Trying to login with your own telegram account should give 'Not Authorized!`
5. Add to the DB a row with your own name and telegram username.
6. Try to login, you should get the jwt token.
